simple_app/app_simple.py

```diff
@@ -408,6 +426,7 @@ def search():
     if query is None:
         query = ''
     db = SimpleDB(db_file, connection_arguments={'check_same_thread': False})  # open database
+    # FIXME: FYI David, doing the to_pandas method because giving fmt='pandas' produces df without col names
     results = db.search_object(query, fmt='astropy')  # get the results for that object
     results: Union[pd.DataFrame, None] = results.to_pandas()  # convert to pandas from astropy table
```

Comment:
FAO @dr-rodriguez

Reply:
Opened https://github.com/dr-rodriguez/AstrodbKit2/issues/39 for this